src/guide/essentials/computed.md

```diff
@@ -328,13 +328,13 @@ export default {
     alwaysSmall: {
       get(previous) {
         if (this.count <= 3) {
-          return this.count;
+          return this.count
         }
 
         return previous;
```

Comment:
sorry for late review (I noticed while implementing chages into translation) - the semicolon remained here